src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/EndreBehandlendeEnhet/EndreBehandlendeEnhet.tsx

```diff
@@ -14,7 +13,7 @@ import { behandendeEnheter } from '../../../../../typer/enhet';
 import { hentFrontendFeilmelding } from '../../../../../utils/ressursUtils';
 
 const EndreBehandlendeEnhet: React.FC = () => {
-    const { åpenBehandling, vurderErLesevisning, erBehandleneEnhetMidlertidig } = useBehandling();
+    const { behandling, vurderErLesevisning, erBehandleneEnhetMidlertidig } = useBehandling();
```

Comment:
Her henter vi behandling fra useBehandling for et menyvalg - kan vi enten sende inn som prop eller bytte alle andre steder som har prop så vi er konsekvente?

Reply:
Fikses i [Bruk context når vi har den](https://github.com/navikt/familie-ba-sak-frontend/pull/2866/commits/6936c65742a2687cbac9e9c6a7bcf42fafef9b11)